oeps/architectural-decisions/oep-0052-arch-event-bus-architecture.rst

```diff
@@ -29,7 +29,7 @@ OEP-0052: Event Bus Architecture
 Overview
 --------
 
-.. note:: Consider this OEP an early draft to provide intent and gather feedback. The language and content of the OEP will change over time as we progress on this work and hammer out certain decisions.
+.. note:: This provisional OEP is expected to be updated with references to new ADRs as work progresses.
```

Comment:
I'm hesitant to say we're only going to add ADRs. Maybe `with references to further documentation`?

Reply:
I may drop the note altogether. Would you consider ADRs just adding clarity? I wasn’t sure, so I called them out. Other updates are already fine.

Reply:
I like the new "expected to be" language -- which I take to mean, "we might forget, but if you notice it's missing... please add!" :-P